packages/vitest/src/node/cli/cli-config.ts

```diff
@@ -602,8 +602,26 @@ export const cliOptionsConfig: VitestCLIOptions = {
   },
   retry: {
     description:
-      'Retry the test specific number of times if it fails (default: `0`)',
-    argument: '<times>',
+      'Retry configuration. Can be a number or JSON string with advanced options (default: `0`)',
+    argument: '<times|config>',
+    subcommands: {
+      count: {
+        description: 'Number of times to retry the test if it fails (default: 0)',
+        argument: '<times>',
+      },
+      strategy: {
+        description: 'Retry strategy: immediate, test-file, or deferred (default: immediate)',
+        argument: '<strategy>',
+      },
+      delay: {
+        description: 'Delay in milliseconds between retries (default: 0)',
+        argument: '<milliseconds>',
+      },
+      condition: {
+        description: 'Condition for retrying: RegExp pattern or custom function',
```

Comment:
I think this needs an explanation where this pattern is applied. According to types, you can also pass down a string, but this message doesn't mention it